packages/jmix-react-core/.eslintrc.json

```diff
@@ -0,0 +1,82 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "plugins": [
+    "react-hooks",
+    "@typescript-eslint"
+  ],
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 9,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true
+    }
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "indent": [
+      "warn",
+      2
+    ],
+    "ordered-imports": 0,
+    "interface-name": 0,
+    "member-access": 0,
+    "object-literal-sort-keys": 0,
+    "jsx-no-lambda": 0,
+    "max-classes-per-file": 0,
+    "member-ordering": 0,
+    "variable-name": 0,
+    "interface-over-type-literal": 0,
+    "no-unnecessary-initializer": 0,
+    "curly": [
+      1,
+      "multi"
+    ],
+    "no-console": [ // Inherited config forbids all methods, but we want to use e.g. .warn() and .error()
+      2,
+      {
+        "allow": [
+          "warn",
+          "error"
+        ]
+      }
+    ],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
```

Comment:
I think should be an `error`. In super-rare cases when we don't want exhaustive deps we can turn off the rule with a comment.

Reply:
**Resolved**: https://github.com/Haulmont/jmix-frontend/pull/311/commits/a21541fede9704eb43aae90e73d940ceb540367c